BackofficeBundle/Controller/SiteController.php

```diff
@@ -27,7 +29,7 @@ class SiteController extends AbstractAdminController
     public function formAction(Request $request, $siteId)
     {
         $site = $this->get('open_orchestra_model.repository.site')->findOneBySiteId($siteId);
-
+        $aliases = $site->getAliases()->toArray();
```

Comment:
first $aliases load values in datatable, second one, values from form. By this way, i'm able to know if aliases have change. @nicolasThal